tests/master/test_pypi.py

```diff
@@ -225,3 +226,81 @@ def test_pypi_read_client_error():
         events = PyPIEvents()
         with pytest.raises(xmlrpc.client.ProtocolError):
             list(events)
+
+@mock.patch('piwheels.master.pypi.requests')
```

Comment:
Hmmm, I generally much prefer the `with` form of `mock.patch`. With the decorator the connection between the mock and its representation within the test (i.e. the "requests" parameter) is not at all clear.